src/app/lib/utilities/getBrandedImage/index.js

```diff
@@ -1,4 +1,4 @@
 const getBrandedImage = (locator, service) =>
-  `${process.env.SIMORGH_ICHEF_BASE_URL}/news/1024/branded_${service}/${locator}`;
+  `${process.env.SIMORGH_ICHEF_BASE_URL}/ace/standard/1024/branded_${service}/${locator}`;
```

Comment:
Can I confirm that updating this is within the scope of the ticket? (NEWSWORLDSERVICE-1662)

Reply:
Are there any examples of the fully formed URL to see if changing the location to `ace/standard` works?

Reply:
Good point! I don't think it works since it changes from a 404 Forbidden to a 404 Not found. I'll change it back.

Original (forbidden): https://ichef.test.bbci.co.uk/news/1024/branded_igbo/729E/test/_63724392_gettyimages-1098075358.jp

Edit (not found): https://ichef.test.bbci.co.uk/ace/standard/1024/branded_igbo/729E/test/_63724392_gettyimages-1098075358.jp